tools/pythonpkg/tests/fast/spark/test_spark_to_csv.py

```diff
@@ -0,0 +1,199 @@
+import pytest
+import tempfile
+
+import os
+
+_ = pytest.importorskip("duckdb.experimental.spark")
+
+from duckdb.experimental.spark.sql import SparkSession as session
+from duckdb import connect, InvalidInputException, read_csv
+from conftest import NumpyPandas, ArrowPandas
+import pandas._testing as tm
+import datetime
+import csv
+
+
+@pytest.fixture
+def df(spark):
+    simpleData = (
+        ("Java", 4000, 5),
+        ("Python", 4600, 10),
+        ("Scala", 4100, 15),
+        ("Scala", 4500, 15),
+        ("PHP", 3000, 20),
+    )
+    columns = ["CourseName", "fee", "discount"]
+    dataframe = spark.createDataFrame(data=simpleData, schema=columns)
+    yield dataframe
+
+
+class TestSparkToCSV(object):
+    @pytest.mark.parametrize('pandas', [NumpyPandas(), ArrowPandas()])
+    def test_basic_to_csv(self, pandas, spark):
+        temp_file_name = os.path.join(tempfile.mkdtemp(), next(tempfile._get_candidate_names()))
```

Comment:
pytest has a `tmp_path` fixture, which should be used instead of manually using `tempfile`
(example from another test)
```py
    def test_read_csv_glob(self, tmp_path, create_temp_csv):
        file1_path, file2_path = create_temp_csv

        # Use the temporary file paths to read CSV files
        con = duckdb.connect()
        rel = con.read_csv(f'{tmp_path}/file*.csv')
        res = con.sql("select * from rel order by all").fetchall()
        assert res == [(1,), (2,), (3,), (4,), (5,), (6,)]
```

Reply:
Makes sense, fixed in `ff1e65087754405fc15f632c20d2c5bb696e4fcb` and `0f6cc01f0b398f236eaf50bb6be2912efde70f89`